### PR TITLE
comment text box

### PR DESCRIPTION
**Pull request by tonianner**
ammended based on class feedback

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -21,11 +21,10 @@ $(document).ready(function(){
   // show description on restaurant porfile
   $('.open').click(function(){
     $('.showDescription').slideToggle('slow');
-    if($(this).text() == 'Close'){
-      $(this).text('Show');
+    if($(this).text() == '(close)'){
+      $(this).text('(show more)');
     }else{
-      $(this).text('Close');
+      $(this).text('(close)');
```
Comment (dshcheung) [inline app/assets/javascripts/application.js]: if these belongs to restaurant profile. please add these to `restaurant.js`

File: app/views/comments/_box.html.erb [hunk 2]
```diff
@@ -1,5 +1,7 @@
-<% @comments.each do |comment| %>
-  <p>Posted by: <%= comment.user.name%></p>
+<% @comments.reverse_each do |comment| %>
```
Comment (dshcheung) [inline app/views/comments/_box.html.erb]: i think reverse_each has a bug. so please user @comments.reverse.each

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (tonianner) [inline app/assets/javascripts/application.js]: tnx.. amended

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -21,11 +21,10 @@ $(document).ready(function(){
   // show description on restaurant porfile
   $('.open').click(function(){
```
Comment (dshcheung) [inline app/assets/javascripts/application.js]: please use `.on("click", function.....`

File: app/views/comments/_box.html.erb (hunk 2 above)
Comment (tonianner) [inline app/views/comments/_box.html.erb]: tnx and done

File: app/assets/javascripts/sessions.coffee [hunk 4]
```diff
@@ -1,3 +0,0 @@
-# Place all the behaviors and hooks related to the matching controller here.
-# All this logic will automatically be available in application.js.
```
Comment (dshcheung) [inline app/assets/javascripts/sessions.coffee:2]: remove this file

File: app/assets/javascripts/users.coffee [hunk 5]
```diff
@@ -1,3 +0,0 @@
-# Place all the behaviors and hooks related to the matching controller here.
-# All this logic will automatically be available in application.js.
```
Comment (dshcheung) [inline app/assets/javascripts/users.coffee:2]: remove this file

File: app/assets/javascripts/sessions.coffee (hunk 4 above)
Comment (tonianner) [inline app/assets/javascripts/sessions.coffee:2]: :+1:

File: app/assets/javascripts/users.coffee (hunk 5 above)
Comment (tonianner) [inline app/assets/javascripts/users.coffee:2]: :+1:

File: app/assets/javascripts/application.js (hunk 3 above)
Comment (tonianner) [inline app/assets/javascripts/application.js]: tnx

**Pull request merged**